extensions/wikia/Piggyback/Piggyback.i18n.php

```diff
@@ -3,516 +3,126 @@
 $messages = [];
 
 $messages['en'] = [
-	'piggybac-desc' => 'Allows logging on as another user',
 	'piggyback' => 'Piggyback',
-	'piggyback-otherusername' => 'Other username:',
-	'piggyback-nosuchuser' => 'There is no user by the name "$1".
-User names are case sensitive.
-Check your spelling.',
-	'piggyback-wronguser' => '$1 is not your user name',
-	'piggyback-loginprompt' => 'Please re-enter your username and password. Warning: This will run ALL custom user-account CSS and JS on your machine. Be careful!',
-	'piggyback-firstlogout' => 'Please first logout back to your account.',
-	'piggyback-itisyou' => 'You cannot piggyback yourself.',
-	'piggyback-issysopto' => 'You cannot piggyback a user with the right to piggyback.',
-	'action-piggyback' => 'log on as another user',
-	'log-name-piggyback' => 'Piggyback log',
```

Comment:
The log seems to be left in the setup file (for historical reference?), so should these messages stay as well for the log page?